web-client/src/presenter/actions/validatePetitionFromPaperAction.js

```diff
@@ -13,30 +14,36 @@ export const aggregateStatisticsErrors = ({ errors, get }) => {
     newErrorStatistics = [];
     const formStatistics = get(state.form.statistics);
 
-    formStatistics.forEach((formStatistic, index) => {
-      const errorStatistic = errors.statistics.find(s => s.index === index);
-      if (errorStatistic) {
-        if (formStatistic.yearOrPeriod === 'Year') {
-          newErrorStatistics.push({
-            enterAllValues:
-              'Enter year, deficiency amount, and total penalties',
-            index,
-          });
-        } else {
-          newErrorStatistics.push({
-            enterAllValues:
-              'Enter period, deficiency amount, and total penalties',
-            index,
-          });
+    if (formStatistics.length) {
+      formStatistics.forEach((formStatistic, index) => {
+        const errorStatistic = errors.statistics.find(s => s.index === index);
+        if (errorStatistic) {
+          if (formStatistic.yearOrPeriod === 'Year') {
+            newErrorStatistics.push({
+              enterAllValues:
+                'Enter year, deficiency amount, and total penalties',
+              index,
+            });
+          } else {
+            newErrorStatistics.push({
+              enterAllValues:
+                'Enter period, deficiency amount, and total penalties',
+              index,
+            });
+          }
```

Comment:
looks like another opportunity for refactoring...

Reply:
Yeah, I think this could be a quick and easy refactor. Let me try and address that.

Reply:
Updated.